seqr/views/apis/data_manager_api_tests.py

```diff
@@ -743,11 +800,115 @@ def test_load_rna_seq_sample_data(self, mock_logger, mock_open, mock_os):
+            'Project Test Reprocessed Project: loaded 1 record(s)'
+        ]
+        self.assertEqual(response.json()['info'], info)
+        mock_logger.info.assert_called_with('\n'.join(info), self.data_manager_user)
+        db_update = {'dbEntity': 'PhenotypePrioritization', 'numEntities': 2,
+                     'parentEntityIds': {'I000002_na19678', 'I000015_na20885'}, 'updateType': 'bulk_create'}
+        mock_model_logger.info.assert_called_with('create PhenotypePrioritizations', self.data_manager_user, db_update=db_update)
+        saved_data = _get_json_for_models(PhenotypePrioritization.objects.filter(),
+                                          nested_fields=[{'fields': ('individual', 'guid'), 'key': 'individualGuid'}])
+        self.assertListEqual(saved_data, EXPECTED_LIRICAL_DATA)
+
+        # Test uploading new data
+        mock_logger.reset_mock()
+        mock_model_logger.reset_mock()
+        mock_file_iter.return_value = self._join_data(PHENOTYPE_PRIORITIZATION_HEADER + UPDATE_LIRICAL_DATA)
+        response = self.client.post(url, content_type='application/json', data=json.dumps({'file': 'lirical_data.tsv.gz'}))
+        self.assertEqual(response.status_code, 200)
+        info = [
+            'Loaded LIRICAL data from lirical_data.tsv.gz',
+            'Project 1kg project nåme with uniçøde: deleted 1 record(s), loaded 2 record(s)'
+        ]
+        self.assertEqual(response.json()['info'], info)
+        mock_logger.info.assert_called_with('\n'.join(info), self.data_manager_user)
+        mock_model_logger.info.assert_has_calls([
+            mock.call('delete PhenotypePrioritizations', self.data_manager_user, db_update={
+                'dbEntity': 'PhenotypePrioritization', 'numEntities': 1,
+                'parentEntityIds': {'I000002_na19678'}, 'updateType': 'bulk_delete',
+            }),
+            mock.call('create PhenotypePrioritizations', self.data_manager_user,
+                      db_update={'dbEntity': 'PhenotypePrioritization', 'numEntities': 2,
+                     'parentEntityIds': {'I000002_na19678'}, 'updateType': 'bulk_create'}),
+        ])
+        self.maxDiff = None
+        saved_data = _get_json_for_models(PhenotypePrioritization.objects.filter(),
+                                          nested_fields=[{'fields': ('individual', 'guid'), 'key': 'individualGuid'}])
+        self.assertListEqual(saved_data, EXPECTED_UPDATED_LIRICAL_DATA)
+
+
+# Tests for AnVIL access disabled
+class LocalDataManagerAPITest(AuthenticationTestCase, DataManagerAPITest):
+    fixtures = ['users', '1kg_project']
+
+
+# Test for permissions from AnVIL only
+class AnvilDataManagerAPITest(AnvilAuthenticationTestCase, DataManagerAPITest):
+    fixtures = ['users', 'social_auth', '1kg_project']
+
+    @mock.patch('seqr.views.utils.permissions_utils.INTERNAL_NAMESPACES', ['my-seqr-billing'])
+    def test_load_phenotype_prioritization_data(self, *args):
```

Comment:
unless you are actually going to explicitly test different behavior in the anvil and non-anvil cases, there is no point in having separate test cases for them

Reply:
I added the anvil tests to cover a branch in the new `is_internal_project` function. Since we remove the function, there is no need for Anvil test cases.